docs/languages.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Languages
+---
+
+## Overview
+
+For each language we require a specific structure for your Dockerfile to be able to interpret it and make it run on our platform
+
+
+The Dockerfiles are language specific just like this one:
+
+```dockerfile
+FROM node:8.9.4
+
+RUN mkdir -p app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN npm install
+
+ENV PORT=80
+
+EXPOSE 80
+
+ENTRYPOINT ["npm","run","start"]
+```
+
+However you will find a common behavior:
+
+* Some are using multi-stage build with a base image (with all the dependencies) and a lighter runtime image to run the app
+* You need to export your compiled code or source files in the folder named `app`
```

Comment:
Maybe emphasise that you can use any build system or tool you wish, the below is just examples, but the key is the output must be in /app (give an example of maven vs grade or something that shows this as  PRO tip kind of thing.)